js/src/forum/components/TextEditor.js

```diff
@@ -122,6 +122,8 @@ export default class TextEditor extends Component {
   setSelectionRange(start, end) {
     const $textarea = this.$('textarea');
 
+    if (!$textarea[0]) return;
```

Comment:
@datitisev wouldn't a length check make more sense?

Aside from that, feel free to merge this.